frontends/web/src/locales/en/app.json

```diff
@@ -222,7 +230,9 @@
       "restore": "I want to restore my wallet from a backup.",
       "restoreMicroSD": "Restore from microSD card",
       "restoreMnemonic": "Restore from mnemonic",
-      "title": "Setup your BitBox02"
+      "title": "Setup your BitBox02",
+      "createTitle": "Create",
+      "restoreTitle": "Restore"
```

Comment:
button.create and button.restore already have the same text, let's just use that instead of adding another field to translate